opentracing-cdi-example/README.md

```diff
@@ -0,0 +1,69 @@
+# OpenTracing CDI - Example
+
+## The example
+
+This example shows the different ways to use the OpenTracing CDI integration. It features
+getting a `Tracer` injected into your components, as well as a `SpanContext` (when the Servlet
+integration is also in place) and an `ActiveSpan` for non-async applications.
+
+The example has the following call tree:
+
+![Trace example](call-tree.png)
+
+Refer to each individual example for more information on how it works.
+
+## Running
+
+In order to visualize the spans, you'll need an instance of Jaeger running locally.
```

Comment:
Even for the examples? Would it be OK if I changed this to mention that the examples can also be run with *any* other OpenTracing-compatible tracers?